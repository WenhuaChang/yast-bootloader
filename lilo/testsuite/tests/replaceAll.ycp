{
    include "testsuite.ycp";
    include "lilo/routines.ycp";

    TEST(``(replaceAll("a", "a", "aaa")), [], nil);
    TEST(``(replaceAll("", "a", "aaa")), [], nil);
    TEST(``(replaceAll("aa", "a", "aaa")), [], nil);
    TEST(``(replaceAll("pepep", "pep", "x")), [], nil);
    TEST(``(replaceAll("abcd", "e", "ASDASD")), [], nil);
    TEST(``(replaceAll("ABCD", "", "X")), [], nil);
    TEST(``(replaceAll("", "A", "X")), [], nil);
    
}
