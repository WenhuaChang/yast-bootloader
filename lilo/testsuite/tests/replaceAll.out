Return	aaa
Return	
Return	aaaaaa
Return	xep
Return	abcd
Return	ABCD
Return	
